.github/dependabot.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "gomod"
```

Comment:
And for all submodules of type `gomod` (replace the submodule name in the prefix):

```
    commit-message:
      prefix: "cert+mod"
```

Reply:
Done